### PR TITLE
Resize

### PR DESCRIPTION
**Pull request by Jopyth**

### CODE REVIEW

File: iywstbdywdiy/sheeper.py [hunk 1]
```diff
@@ -88,24 +89,24 @@ def make_image(self, image, is_test, bounding_boxes=(), stamps=()):
...
 
-            resized_to_bb = stamps[i].resize((width, height), Image.ANTIALIAS)
+            resized_to_bb = stamps[i].resize((width, height), Image.LANCZOS)
 
             layer = Image.new("RGBA", image.size, (0, 0, 0, 0))
             layer.paste(resized_to_bb, box=(x1, y1))
             out = Image.alpha_composite(out, layer)
 
-        out.convert("RGB").save(output_path)
+        out.convert("RGB").save(output_path, quality=95)
```
Comment (Bartzi) [inline iywstbdywdiy/sheeper.py]: why not `quality=100`?

File: iywstbdywdiy/sheeper.py [hunk 2]
```diff
@@ -8,11 +8,12 @@
 
 
 class Generator:
-    def __init__(self, output, resize_max=500, search_path=None):
+    def __init__(self, output, resize_max=500, search_path=None, json_path=None):
         self.resize_max = resize_max
         self.test_stamps = []
         self.train_stamps = []
         self.output = output
+        self.json_output = output if json_path is None else json_path
```
Comment (Bartzi) [inline iywstbdywdiy/sheeper.py]: we could rename `output` and `json_output` to `output_path` and `json_output_path`, couldn't we?

File: iywstbdywdiy/sheeper.py (hunk 1 above)
Comment (Jopyth) [reply iywstbdywdiy/sheeper.py]: [From the docs](http://pillow.readthedocs.io/en/5.1.x/handbook/image-file-formats.html#jpeg):
> The image quality, on a scale from 1 (worst) to 95 (best). The default is 75. **Values above 95 should be avoided**; 100 disables portions of the JPEG compression algorithm, and results in **large files with hardly any gain in image quality**.

File: iywstbdywdiy/sheeper.py (hunk 2 above)
Comment (Jopyth) [reply iywstbdywdiy/sheeper.py]: This makes the command line arguments quite long, but might still be good.

**Pull request merged**